modules/homefinder/widgets/homefinder-widgets.js

```diff
@@ -0,0 +1,33 @@
+(function () {
+	hf_data.forEach(function(data, index) {
+		if ( data.type === 'directory' ) {
+			var widget = new HomeFinder.Widgets.AdvertiserDirectory({
```

Comment:
__[jshint]__
* 'widget' is defined but never used.
* 'HomeFinder' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->